### PR TITLE
buffer flow

### PR DESCRIPTION
**Pull request by ppclunghe**
Brief comments on the purpose of your changes:
After Lido V2 upgrade we want to make the bunch of changes in accounting model, in this update we add ETH inflow/outflow for protocol buffer
Made this PR instead of 3372

**For Dune Engine V2**

I've checked that:

### General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [x] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [x] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [x] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [x] `coin_id` represents the ID of the coin on coinpaprika.com
* [x] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [x] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [x] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [x] where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: models/lido/accounting/ethereum/lido_accounting_ethereum_buffer_inflow.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{{ config(
+        alias ='buffer_inflow',
+        partition_by = ['period'],
```
Comment (jeff-dude) [inline models/lido/accounting/ethereum/lido_accounting_ethereum_buffer_inflow.sql:3]: if i'm reading the spell correctly, this is partitioning by `evt_block_time`? that is much too granular for partitions. i'd suggest either removing this config option or introducing a new column in the select below that is at minimum day (or week, month, etc) to partition by

File: models/lido/accounting/ethereum/lido_accounting_ethereum_buffer_outflow.sql [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{{ config(
+        alias ='buffer_outflow',
+        partition_by = ['period'],
```
Comment (jeff-dude) [inline models/lido/accounting/ethereum/lido_accounting_ethereum_buffer_outflow.sql:3]: same idea here

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/lido/accounting/ethereum/lido_accounting_ethereum_buffer_inflow.sql (hunk 1 above)
Comment (ppclunghe) [reply models/lido/accounting/ethereum/lido_accounting_ethereum_buffer_inflow.sql]: thank you for review, fixed config with new "day" column

Comment (jeff-dude) [reply models/lido/accounting/ethereum/lido_accounting_ethereum_buffer_inflow.sql]: great! we'll let the test run, then hopefully all set

Comment (gregshestakovlido) [reply models/lido/accounting/ethereum/lido_accounting_ethereum_buffer_inflow.sql]: It seems that test execution takes too long and fails after 90 minutes. @jeff-dude, maybe removing partition will help here or it is something in test flow?

Comment (jeff-dude) [reply models/lido/accounting/ethereum/lido_accounting_ethereum_buffer_inflow.sql]: i don't think it's due to your spells in the PR. for some reason, our gh action ci test is picking up more models than it should and causing longer runtimes, then timeouts. i'm trying to look into that now. i would assume no changes here, just waiting for resolution there and a successful run after

**Review by jeff-dude — APPROVED** [review]

File: models/lido/accounting/ethereum/lido_accounting_ethereum_buffer_inflow.sql (hunk 1 above)
Comment (jeff-dude) [reply models/lido/accounting/ethereum/lido_accounting_ethereum_buffer_inflow.sql]: all good, unrelated issue resolved

**Pull request merged**